README-bn.md

```diff
@@ -0,0 +1,197 @@
+
+স্থানীয়ভাবে সাইট তৈরি এবং পরীক্ষা করতে, চালান:
+
+```bash
+# install dependencies
+npm ci
+make serve
+```
+
+এটি পোর্ট 1313-এ স্থানীয় হুগো সার্ভার শুরু করবে। ওয়েবসাইট দেখতে <http://localhost:1313>-এ আপনার ব্রাউজার খুলুন। আপনি সোর্স ফাইলগুলিতে পরিবর্তন করার সাথে সাথে, Hugo ওয়েবসাইট আপডেট হবে এবং একটি ব্রাউজার রিফ্রেশ করতে বাধ্য করে।
+
+
+## API রেফারেন্স পৃষ্ঠা (Page) তৈরি করা
+
+
+`content/en/docs/reference/kubernetes-api` এ অবস্থিত API রেফারেন্স পৃষ্ঠাগুলি <https://github.com/kubernetes-sigs/reference-docs/tree/master/gen-resourcesdocs> ব্যবহার করে Swagger স্পেসিফিকেশন থেকে তৈরি করা হয়েছে
+
+একটি নতুন কুবারনেটিস(Kubernetes) রিলিজের জন্য রেফারেন্স পৃষ্ঠাগুলি আপডেট করতে এই পদক্ষেপগুলি অনুসরণ করুন:
+
+1. `api-ref-generator` সাবমডিউল পুল (Pull) করুন:
+
+   ```bash
+   git submodule update --init --recursive --depth 1
+   ```
+
+2. Swagger স্পেসিফিকেশন আপডেট করুন:
+
+   ```bash
+   curl 'https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json' > api-ref-assets/api/swagger.json
+   ```
+
+3. `api-ref-assets/config/`-এ, নতুন রিলিজের পরিবর্তনগুলি প্রতিফলিত করতে `toc.yaml` এবং `fields.yaml` ফাইলগুলিকে  হালনাগাদ  করে নিন।
+
+
+4. পরবর্তী, পৃষ্ঠাগুলি তৈরি করুন:
+
+   ```bash
+   make api-reference
+   ```
+
+  আপনি একটি কনটেইনার চিত্র (Container image) থেকে সাইট তৈরি এবং পরিবেশন করে স্থানীয়ভাবে ফলাফল পরীক্ষা করতে পারেন:
+
+   ```bash
+   make container-image
+   make container-serve
+   ```
+
+   একটি ওয়েব ব্রাউজারে, API রেফারেন্স দেখতে <http://localhost:1313/docs/reference/kubernetes-api/> এ যান।
+
+5.যখন নতুন চুক্তির সমস্ত পরিবর্তন কনফিগারেশন ফাইল `toc.yaml` এবং `fields.yaml`-এ প্রতিফলিত হয়, তখন নতুন জেনারেট করা API রেফারেন্স পৃষ্ঠাগুলির সাথে একটি পুল অনুরোধ (Pull request) তৈরি করুন।
+
+
+## সমস্যা সমাধান
+
+### ত্রুটি: failed to transform resource: TOCSS: failed to transform "scss/main.scss" (text/x-scss): এই বৈশিষ্ট্যটি আপনার বর্তমান Hugo সংস্করণে উপস্থিত নয়৷
+
+
+প্রযুক্তিগত কারণে হুগোকে দুই সেট বাইনারিতে পাঠানো হয়েছে। বর্তমান ওয়েবসাইটটি শুধুমাত্র **Hugo Extended** সংস্করণের উপর ভিত্তি করে চলে। [রিলিজ পৃষ্ঠা](https://github.com/gohugoio/hugo/releases) নামের মধ্যে `বর্ধিত(extended)` সহ সংরক্ষণাগারগুলি খুঁজুন। নিশ্চিত করতে, `হুগো সংস্করণ (hugo version)` চালান এবং `বর্ধিত (extended)` শব্দটি সন্ধান করুন।
+
+
+### অনেকগুলি খোলা ফাইলের জন্য macOS সমস্যা সমাধান করা হচ্ছে
+
+আপনি যদি macOS-এ `make serve` চালান এবং নিম্নলিখিত ত্রুটি পান:
+
+```bash
+ERROR 2020/08/01 19:09:18 Error: listen tcp 127.0.0.1:1313: socket: too many open files
+make: *** [serve] Error 1
+```
+
+খোলা ফাইলগুলির জন্য বর্তমান সীমা পরীক্ষা করার চেষ্টা করুন:
+
+`launchctl limit maxfiles`
+
+তারপর নিম্নলিখিত কমান্ডগুলি চালান (<https://gist.github.com/tombigel/d503800a282fcadbee14b537735d202c> থেকে নেয়া):
+
+```shell
+#!/bin/sh
+
+# নিচের গুলি মূল সারাংশ(gist) লিঙ্ক, এবং আমার সারাংশ(gist) লিঙ্ক এর সাথে সম্পর্ক স্থাপন করি.
```

Comment:
This looks like a good example of a comment that has been localized.

Reply:
okay! 😊